tools/clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -7772,10 +7772,10 @@ def err_hlsl_unsupported_for_version_lower : Error<
 def err_hlsl_unsupported_keyword_for_min_precision : Error<
    "%0 is only supported with -enable-16bit-types option">;
 def warn_hlsl_derivatives_in_wrong_shader_kind: Warning<
-   "Intrinsic %0 requires derivatives - only available in pixel, compute, amplification, mesh, or broadcast node shaders.">,
+   "Intrinsic %0 called by %1 requires derivatives - only available in pixel, compute, amplification, mesh, or broadcast node shaders.">,
```

Comment:
I feel like ", potentially used by %1, " might be more appropriate than "called by", since called by may not be technically accurate.

Reply:
Updated.